examples/plot_choose_thumbnail.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
 """
+.. _choose-thumbnail-example:
```

Comment:
You don't need to do that, references are created automatically for examples. Don't remember the exact reference naming though, you'll have to browse the doc and grep for something that contains `sphx_glr`